main.go

```diff
@@ -504,6 +509,7 @@ func transformCompile(args []string) ([]string, error) {
 			// messy.
 			name = "_cgo_" + name
 		default:
+			file = transformLineInfo(i, file)
 			file = transformGo(file, info, blacklist)
 			name = fmt.Sprintf("z%d.go", i)
```

Comment:
Do we still need this?

Reply:
No

Reply:
Personally, I like the idea of transforming line info on every package/file  *only* when `-tiny` is passed. Otherwise only packages in GOPRIVATE would have their line info altered. This may be up for discussion, but if the aim of `-tiny` really is to reduce the size of the binary, that would certainly help, because statistically most packages will probably not be in GOPRIVATE.